servers/apps/test/shm_test.c

```diff
@@ -0,0 +1,84 @@
+#include <resea/ipc.h>
+#include <resea/printf.h>
+#include <resea/syscall.h>
+#include <resea/task.h>
+#include <string.h>
+#include "test.h"
+
+#define MSG "hello task b"
+
+void test() {
+    struct message m;
+    error_t err;
+
+    // create shared memory
```

Comment:
Nit: In Resea, comments should be `// A sentence.` for consistency. For example: `// Create a shared memory.`.